src/Nethermind/Nethermind.Serialization.Rlp/RlpDecoderExtensions.cs

```diff
@@ -33,6 +33,18 @@ public static T[] DecodeArray<T>(this IRlpValueDecoder<T> decoder, ref Rlp.Value
             return result;
         }
 
+        public static T[] EncodeArray<T>(this IRlpStreamDecoder<T> decoder, RlpStream rlpStream, RlpBehaviors rlpBehaviors = RlpBehaviors.None)
```

Comment:
This is `Decode`, not `Encode`

Reply:
think this was a leftover from a refactor, its removed